openapi/db/path.py

```diff
@@ -27,6 +27,25 @@ def db(self) -> CrudDB:
     def db_table(self):
         return self.db.metadata.tables[self.table]
 
+    async def get_total(
+            self, *, filters=None, query=None, table=None,
+            query_schema='query_schema', conn=None
+    ):
```

Comment:
I don't think this should be a separate method.
It needs to come out from the query building so we know it is exactly the same query like the one we extract data from.